src/main/java/com/couchbase/lite/CouchbaseLite.java

```diff
@@ -98,4 +103,22 @@ private static String verifyDir(@NonNull File dir) {
 
         throw new IllegalStateException("Cannot create or access temp directory at " + path);
     }
+
+    @NonNull
+    private static Map<String, String> loadErrorMessages() {
+        final Map<String, String> errorMessages = new HashMap<>();
+
+        try (InputStream is = CouchbaseLite.class.getResourceAsStream(ERRORS_PROPERTIES_PATH)){
+            if (is == null) { throw new FileNotFoundException("Cannot find resource at " + ERRORS_PROPERTIES_PATH); }
+            final Properties errors = new Properties();
+            errors.load(is);
+            for (final String name: errors.stringPropertyNames()) {
```

Comment:
You can avoid the copy by saying:
return (Map<String, String>) (Map) errors;

As long as you comment it, and put an @SuppressWarnings annotation on it, this might be a better solution...